sources/osg/Light.js

```diff
@@ -199,6 +199,43 @@ define( [
             return this._ground;
         },
 
+        computeAttenuation: function ( dist ) {
+            // formula playground
+            // https://www.desmos.com/calculator/nmnaud1hrw
+            var constant = this._attenuation[ 0 ];
+            var linear = this._attenuation[ 1 ] * dist;
+            var quadratic = this._attenuation[ 2 ] * dist * dist;
+            return ( 1.0 / ( constant + linear + quadratic ) );
+        },
+        // First step to light intersection
+        // for culling
+        checkForNearerLightFar: function ( currentFar, epsilon ) {
```

Comment:
Could be either in osgShadow/shadowMap  (already a lot full) or a osgShadow/shadowFarCompute ?

(or we could separate shadowMap into shadowMap and a ShadowFrustumAlgo, allowing for multiple algo basic bbox, enhanced bbox, atlas CSM, etc... and incremental update without breaking existing)